fluentlenium-core/src/main/java/org/fluentlenium/core/inject/ContainerFluentControl.java

```diff
@@ -46,6 +48,16 @@ public ContainerFluentControl(FluentControl adapterControl) {
         this.adapterControl = adapterControl;
     }
 
+    /**
+     * Make control over chromium api.
+     *
+     * @return a new chromium api instance
+     * @param driver remote webdriver instance
+     */
+    public final ChromiumApi makeChromiumApi(RemoteWebDriver driver) {
+        return new ChromiumApi(driver);
```

Comment:
Please use existing driver, there shouldn't be any need to provide remoteWebDriver as a parameter

Reply:
Fixed.